pkg/apis/serving/v1alpha2/framework_pytorch.go

```diff
@@ -32,6 +33,11 @@ func (s *PyTorchSpec) GetStorageUri() string {
 	return s.StorageURI
 }
 
+func (s *PyTorchSpec) GetResourceRequirement() *v1.ResourceRequirements {
```

Comment:
GetContainer().Resources should achieve the same effect in a cleaner way.

Reply:
Good suggestion. I will update code this way and we can avoid modifying each predictor's code one by one!

Reply:
Tried but the GetContainer function require modelName parameter which only available in metadata. If we are going to use that we need change validateEndpoint in inferenceservice_validation.go to take metadata as parameter, I think it's not worth doing so keep the original design. Please take a look, thanks.